router/batchrouter/asyncdestinationmanager/marketo-bulk-upload/marketobulkupload.go

```diff
@@ -96,9 +97,9 @@ func (b *MarketoBulkUploader) Poll(pollInput common.AsyncPoll) common.PollStatus
 
 	if asyncResponse.Error != "" {
 		b.logger.Errorw("[Batch Router] Failed to fetch status for",
```

Comment:
Shouldn't you be using the non-sugared logger? i.e. `Errorn` and not `Errorw`?

Reply:
Yeah, wanted to move to the non-sugared logger as well but the changes will be more. I will create a separate PR for it.

Reply:
Did you see how the logger actually works though?

https://github.com/rudderlabs/rudder-go-kit/blob/main/logger/field.go#L113

If you want to use the field type with the sugared logger then you should use `Expand`. I suggest changing all in this PR so that you don't have to use `Expand` and then remove it when you migrate to `Errorn`.